### PR TITLE
#17 Feature: add --open option

### PR DESCRIPTION
**Pull request by peetwriter**

### CODE REVIEW

**Comment by peetwriter** [comment]
Hello @ranyitz , I've added --open feature for getAciton with tests, but not sure if tests are written in the right way - I'm mocking called function to check if they were called with correct parameters, but could not test prompts params in the right way. Please, check if it is ok with you.

File: src/actions/get.js [hunk 1]
```diff
@@ -13,5 +16,33 @@ module.exports = (workspace, name, options = {}) => {
     throw new NotFoundModuleError(name, suggestions);
   }
 
+  if (open) {
+    const opnOptions = { wait: false };
+    if (moduleOccurrences.length === 1) {
+      const { nodeModulesPath, name: moduleName } = moduleOccurrences[0];
+      const packagePath = `${nodeModulesPath}/${moduleName}`;
```
Comment (ranyitz) [inline src/actions/get.js]: Can you use `path.join` instead of template string?

File: src/actions/get.js [hunk 2]
```diff
@@ -13,5 +16,33 @@ module.exports = (workspace, name, options = {}) => {
...
+      opn(packagePath, opnOptions);
+    } else {
+      const listItems = [];
+      moduleOccurrences.forEach(item => {
+        const { nodeModulesPath, version, parent, name: moduleName } = item;
+        listItems.push({
+          title: `${version} ${parent ? parent.name : 'node_modules'}`,
+          value: `${nodeModulesPath}/${moduleName}`,
```
Comment (ranyitz) [inline src/actions/get.js]: also here (`path.join`)

File: src/cli.js [hunk 3]
```diff
@@ -19,7 +19,8 @@ try {
       '-w, --why',
       'add information regarding why this package was installed',
     )
-    .option('-d, --debug', 'see full error messages, mostly for debugging');
+    .option('-d, --debug', 'see full error messages, mostly for debugging')
+    .option('-o, --open', '"open editor at the module\'s directory"');
```
Comment (ranyitz) [inline src/cli.js]: why the double `'" "'`?

**Comment by peetwriter** [comment]
Hello @ranyitz, thank you for your comments - I've fixed them all - except the one with open in default Editor. I've added an option to opn function : `app: process.env.EDITOR` but it will only work if Editor is set in user's environment variables. It is the rare case among users. There are also some options to make ---open option open an editor like: 
- open a specific file (package.json for example) - but it will now open the default user editor but a default editor for this type of a file.
- explicitly indicate what editor to use (like Atom or VsCode).
Let me know what solution will work for you.

**Comment by ranyitz** [comment]
@peetwriter Thanks a lot! 🎉 
I did some changes to the PR.

I'm not sure how many people have `process.env.EDITOR` configured.
Opening `package.json` seems to be a better idea as the user would probably wanted to see it in the first place.

I've added it to the main readme, and change the output and tests a bit.

**Pull request merged**